src/bin/server/web.rs

```diff
@@ -0,0 +1,27 @@
+use actix_web::{HttpRequest, HttpResponse};
+use mime_guess::guess_mime_type;
+
+#[derive(RustEmbed)]
+#[folder = "web/"]
+struct WebAssets;
+
+/// Get a file from the embedded web assets
+fn handle_embedded_file(path: &str) -> HttpResponse {
+    match WebAssets::get(path) {
+        Some(content) => HttpResponse::Ok()
+            .content_type(guess_mime_type(path).to_string())
```

Comment:
Other usages of `guess_mime_type` are done through `mime_guess::guess_mime_type`. Please use the full name with module.